subsys/bluetooth/mesh/light_ctl_srv.c

```diff
@@ -378,7 +391,6 @@ static void temp_set(struct bt_mesh_light_temp_srv *srv,
 	ctl_srv->handlers->set(ctl_srv, ctx, &gen_set, &gen_rsp);
 
 	store_state(ctl_srv);
-	light_ctl_pub(ctl_srv, &gen_rsp);
```

Comment:
We should publish here, as this is a state change. Here and below.

Reply:
I've restored publish functions back.